src/firefly/js/visualize/ui/ImageSearchPanelV2.jsx

```diff
@@ -109,11 +109,12 @@ function getContexInfo(renderTreeId) {
 
 function ImageSearchPanel({resizable=true, onSubmit, gridSupport = false, multiSelect, submitText, onCancel=dispatchHideDropDown}) {
     const archiveName =  get(getAppOptions(), 'ImageSearch.archiveName');
-    const resize = resizable ? {resize: 'both', overflow: 'hidden', paddingBottom: 5} : {};
+    const resize = {resize: 'both', overflow: 'hidden', paddingBottom: 5};
```

Comment:
I think the image search panel needs a limit to how widget. I played with it and
```
max-width: 900px;
```
seemed to work nice.